<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="voice_btn">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>770</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#voice_btn:focus{border-radius:10px;background-color: rgb(255, 255, 255);border: 1px solid #E8E8E8;color: rgb(45, 60, 99)}
QPushButton#voice_btn{border-radius:10px;background-color: rgb(255, 255, 255);border: 1px solid #E8E8E8;color: rgb(45, 60, 99)}</string>
   </property>
   <property name="text">
    <string>开始语音识别</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>1761</width>
     <height>691</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="title">
    <string>光照传感器</string>
   </property>
   <widget class="QTextBrowser" name="light_textBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>1701</width>
      <height>651</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_info">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>910</y>
     <width>1331</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="light_btn">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>770</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>20</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#light_btn:focus{border-radius:10px;background-color: rgb(255, 255, 255);border: 1px solid #E8E8E8;color: rgb(45, 60, 99)}
QPushButton#light_btn{border-radius:10px;background-color: rgb(255, 255, 255);border: 1px solid #E8E8E8;color: rgb(45, 60, 99)}</string>
   </property>
   <property name="text">
    <string>获取光照度</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
